<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBindBoatBox</class>
 <widget class="QDialog" name="DialogBindBoatBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boat-Box einer Zille Zuordnen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_infoText">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Unzugeordnete Boat-Box ist vor 4s 
durch das Start-Tor gefahren!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Zillen-Nummer</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Bitte Zillen-Nummer ausw채hlen:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_boatName">
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_replacementBoat">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Ersatzzille f체r</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
        <item>
         <widget class="QLabel" name="label_infoText_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Statt welcher Zille 
f채hrt diese Ersatzzille?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_boatID">
          <property name="maxVisibleItems">
           <number>20</number>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_later">
       <property name="text">
        <string>Sp채ter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
